sherpa/astro/datastack/tests/test_datastack.py

```diff
@@ -680,7 +680,7 @@ def test_case_7(self):
 
         assert f == [1, 2]
 
-        f = datastack.query_by_obsid('7867')
+        f = datastack.query_by_obsid(7867)
```

Comment:
Ahhh, looking at the test as a whole I see there's two calls to this method, and we now test both cases. Phew and sorry for the false alarm.